nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -277,6 +277,8 @@ public Config getConfig() {
    */
   private final Map<ExpressionTree, Nullness> computedNullnessMap = new LinkedHashMap<>();
 
+  private GenericsChecks genericsChecks = new GenericsChecks();
```

Comment:
In a follow up (not for this PR), we should probably make a `Config` field inside `GenericsChecks` and then stop passing it as a parameter to a whole bunch of methods.  Again not for this PR